packaging/rpm/publish-packages.sh

```diff
@@ -7,6 +7,9 @@ if [ -z "${REPOSITORY_URL}" ]; then
     exit 1
 fi
 
+# The rpmbuild command creates the package files in the RPMS directory.
+cd /workspace/build/RPMS
```

Comment:
Same here just check in the RPMS. Note there `<arch>` and `noarch` folders in RPMS. I didn't add `-depth` because there isn't anything else there.